org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/preferences/SonarLintPreferencesInitializer.java

```diff
@@ -32,7 +32,9 @@ public class SonarLintPreferencesInitializer extends AbstractPreferenceInitializ
   public void initializeDefaultPreferences() {
     var node = DefaultScope.INSTANCE.getNode(SonarLintUiPlugin.PLUGIN_ID);
     node.put(SonarLintConsole.P_SHOW_CONSOLE, SonarLintConsole.P_SHOW_CONSOLE_ON_ERROR);
+    node.putBoolean(SonarLintConsole.P_VERBOSE_OUTPUT, false);
```

Comment:
What was happening before this change? verbose output was enabled by default?

Reply:
This is not linked to the UI. This is for the preferences saved on disk as Eclipse does preserve them this way.
E.g. when exporting the preferences of the workspace (the settings) to share them with others, this is used to fill in the default values.
When the other one was added some time ago, I assume this was just forgotten.